content/articles/importing-external-files-using-webpack-loaders-the-beginners-guide/index.md

```diff
@@ -0,0 +1,291 @@
+### Importing External Files using Webpack Loaders: The Beginners Guide 
+In this article,  we will learn about Webpack and how to use Webpack loaders to import external files while making use of Webpack. 
+
+Webpack may not be straighforward at first, but you will get the hang of it after working with it for some time. 
+
+This tutorial is tailored to suit the needs of a beginner. As a prerequisite, your understanding of HTML, CSS, JavaScript and the use of the Terminal or a Command Line Interface (CLI) will be helpful. 
+
+### Table Of Contents
+1. What is Webpack?
+2. Installing Webpack
+3. Loaders and Plugins
+4. Making use of Webpack loaders to import various types of files
+
+Let us dive in!
+
+### What is Webpack?
+Webpack is a tool that allows you to modularise your code and helps you to push your codes to one file in other for you not to worry about the hierarchy of your files, especially when working with multiple JavaScript files. 
```

Comment:
Explain more about it.

What if we don't use webpack? How's it better? Where do we use it? Why to use it?